src/v3/src/transformer/oktaVerify/transformOktaVerifyChannelSelection.ts

```diff
@@ -62,11 +62,19 @@ export const transformOktaVerifyChannelSelection: IdxStepTransformer = ({
     'authenticator.channel',
     formBag.uischema.elements as UISchemaElement[],
   ) as FieldElement;
-  channelSelectionElement.label = loc('oie.enroll.okta_verify.select.channel.description', 'login');
+  channelSelectionElement.label = loc('oie.enroll.okta_verify.select.channel.subtitle', 'login');
   channelSelectionElement.options.format = 'radio';
   const { options: { inputMeta: { options = [] } } } = channelSelectionElement;
   channelSelectionElement.options.customOptions = options
-    .filter((opt) => opt.value !== lastSelectedChannel)
+    .filter((opt) => {
```

Comment:
Is it possible for `samedevice` to be the only enrollment option? If so, `channelSelectionElement.options.customOptions[0].value` below will fail

Reply:
it is possible for `samedevice` to be the only channel, but that is only when OV security setting is set to HIGH.  In that case, the user would not have to option to navigate to this channel selection screen anyway, so It probably wouldnt be a problem.  But I have added a safeguard if condition just to be safe.